src/docs/blog/typesafe_configuration/post/typesafe.kt

```diff
@@ -17,9 +17,12 @@ data class Host(val value: String)
 
 val env = Environment.ENV
 
-val portLens: Lens<Environment, Port> = EnvironmentKey.int().map(::Port).defaulted("PORT", Port(9000))
-val timeoutLens: Lens<Environment, Timeout?> = EnvironmentKey.duration().map(::Timeout).optional("TIMEOUT")
-val hostsLens: Lens<Environment, List<Host>> = EnvironmentKey.map(::Host).multi.required("HOSTNAMES")
+val portLens: Lens<Environment, Port> =
+    EnvironmentKey.int().map(::Port).defaulted("PORT", Port(9000))
```

Comment:
We could also go for 
```kotlin
val portLens: Lens<Environment, Port> = EnvironmentKey.int()
    .map(::Port)
    .defaulted("PORT", Port(9000))
```
But, if I remember correctly, the rest of the code base is not splitting lines after each method call, right?